libraries/DataFlash/DataFlash_MAVLinkLogTransfer.cpp

```diff
@@ -66,6 +66,11 @@ void DataFlash_Class::handle_log_message(GCS_MAVLINK &link, mavlink_message_t *m
  */
 void DataFlash_Class::handle_log_request_list(GCS_MAVLINK &link, mavlink_message_t *msg)
 {
+    if (_log_sending_chan >= 0) {
+        gcs().send_text(MAV_SEVERITY_INFO, "In progress");
```

Comment:
I think this is too short (what's in progress?) and I think that sending the message just to the GCS that requested the log would be nicer - we already have the `link` here so it is easy.